File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/advanced_search_request.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class AdvancedSearchRequest:
    keyword: str = None
    type: str = None
    filters: Dict[str, Any] = None
    sort_field: str = None
    sort_direction: str = None
    page: int = None
    size: int = None

File: sdkwork-app-sdk-go/types/advanced_search_request.go
package types


type AdvancedSearchRequest struct {
	Keyword string `json:"keyword"`
	Type string `json:"type"`
	Filters map[string]interface{} `json:"filters"`
	SortField string `json:"sortField"`
	SortDirection string `json:"sortDirection"`
	Page int `json:"page"`
	Size int `json:"size"`
}
